docs/installation/ca/installing-ca-clone-with-ldaps-using-bootstrap-ds-certs.adoc

```diff
@@ -0,0 +1,185 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="installing-ca-clone-with-ldaps-using-bootstrap-ds-certs_{context}"]
+= Installing CA Clone with LDAPS Using Bootstrap DS Certs
+
+Follow this process to install a CA subsystem as clone of an existing CA subsystem that runs with a bootstrap SSL server certificate.
+
+Prior to installation, please ensure that the link:../others/installation-prerequisites.adoc[Installation Prerequisites] are configured.
+
+== DS Configuration 
+
+Once the prerequisites listed above are completed on the clone system, go on the existing system and export the DS signing certificate into `ds_signing.p12` and copy the certificate into clone system with the following command:
+
+[literal,subs="+quotes,verbatim"]
+....
+$ pki -d /etc/dirsrv/slapd-localhost \
+-C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+pkcs12-export --pkcs12-file ds_signing.p12 \
+--pkcs12-password Secret.123 Self-Signed-CA
+....
+Import the `ds_signing.p12` into the clone DS instance with the following command:
+
+[literal,subs="+quotes,verbatim"]
+....
+$ pki -d /etc/dirsrv/slapd-localhost \
+-C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+pkcs12-import --pkcs12-file ds_signing.p12 \
+--pkcs12-password Secret.123
+....
+In the DS clone, create a temporary DS server certificate as described in xref:../others/enabling-ssl-connection-in-ds-with-temp-cert.adoc[Creating DS Server Certificate].
+Note that the certificate subject DN should match the clone's hostname ( i.e `--subject "CN=secondary.example.com"` ).
+
+Then enable the SSL connection as described in xref:../others/enabling-ssl-connection-in-ds-with-temp-cert.adoc[Enabling SSL Connection].
+
+After the successful DS restart, export the DS Signing Certificate into 'ds_signing.crt' as described in link:../others/exporting-ds-certificates.adoc[Exporting DS Signing Certificate].
+
+Some useful tips:
+
+ - Make sure the firewall on the master allows external access to LDAP from the clone
+ - Make sure the firewall on the clone allows external access to LDAP from the master
```

Comment:
In other docs we've been using the term primary & secondary servers instead of master & clone. We probably should be more consistent, or merge these sentences into something like "make sure the DS firewall allows access from PKI server and other DS instances".

Reply:
sure.  This are remnants from the old md file. I'll replace.  Thanks.